openapi/components/schemas/pastoral-item.yaml

```
@@ -0,0 +1,243 @@
+title: a Pastoral Record
+description: A pastoral record
+readOnly: true
+type: array
+properties:
```

```suggestion
type: array
items:
    properties:
```
it is not displaying the array correctly